core/chain-configs/src/client_config.rs

```diff
@@ -70,7 +70,7 @@ impl GCConfig {
     }
 }
 
-#[derive(Clone, Serialize, Deserialize)]
+#[derive(Clone, Debug, Serialize, Deserialize)]
```

Comment:
Due to another PR merging, please re-implement `Debug` and `Serialize` by implementing them for `MutableConfigValue`.